lib/alephant/broker/environment.rb

```diff
@@ -32,6 +32,14 @@ def path
         settings['PATH_INFO']
       end
 
+      def etag
+        settings["IF_NONE_MATCH"]
+      end
```

Comment:
Hmm, it's semantics, but maybe i'd call this method `if_none_match` as this is a representation of the incoming request